app/src/cli.c

```diff
@@ -697,6 +698,11 @@ static const struct sc_option options[] = {
         .longopt = "rotation",
         .argdesc = "value",
     },
+    {
+        .longopt_id = OPT_ROOT,
+        .longopt = "root",
```

Comment:
Nit: move this above "rotation" to preserve alphabetical order

Reply:
Fixed.